structuredevent-model/src/main/java/com/sequenceiq/cloudbreak/structuredevent/conf/StructuredEventEnablementConfig.java

```diff
@@ -0,0 +1,41 @@
+package com.sequenceiq.cloudbreak.structuredevent.conf;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class StructuredEventEnablementConfig {
+
+    @Value("${cb.kafka.bootstrap.servers:}")
+    private String bootstrapServers;
+
+    @Value("${cb.audit.filepath:}")
+    private String auditFilePath;
+
+    @Value("${audit.service.enabled}")
```

Comment:
would be more flexible with a default value "false" and set explicitly in the enabled services.
because of transitive dependencies, other services that are not explicitly adding :structuredevent-model (like RedBeams) will fail upon startup

Reply:
I am trying to add the. variable to all module